style.css

```diff
@@ -0,0 +1,81 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+main {
```

Comment:
are you sure you will not use main and hr tags for other pages?
If you are not, that will effect your design after appearing new main or new hr

Reply:
each page have style file so not effect